amplify.yml

```
@@ -0,0 +1,89 @@
+    # Next.js built assets - these are automatically hashed, safe for long cache
+    - pattern: "/_next/static/**"
+      headers:
+        - key: Cache-Control
+          value: "public, max-age=31536000, immutable"
+    
+    # Next.js optimized images - 1 year cache (CRITICAL for performance)
+    - pattern: "/_next/image**"
+      headers:
+        - key: Cache-Control
+          value: "public, max-age=31536000, immutable"
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)